Sources/TuistCore/Analytics/BinaryCacheAnalytics.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import Path
+
+public struct BinaryCacheAnalytics {
+    public let hashes: [AbsolutePath: [String: String]]
+    public let cacheItems: [AbsolutePath: [String: CacheItem]]
```

Comment:
I noticed that `CacheItem` has a `hash` attribute. Do they have the same value as the ones in `hashes`? If so, I'd consider consolidating both and even dropping `BinaryCacheAnalytics`. Or are we aligning this with some server schema?

Reply:
No, we can consolidate these two properties into one.